.github/CONTRIBUTING.md

```diff
@@ -303,6 +303,12 @@ webpack is feature rich and documentation is a time sink. We
 greatly appreciate any time spent fixing typos or clarifying sections in the
 documentation.
 
+## Releasing
+
+Run `yarn publish:monorepo` to build all packages and bump versions, this will then get published on npm.
+
+> NOTE: Packages which are internally consumed need to be manually bumped in each package.json file which uses the package.
```

Comment:
Can you clarify? Ideally we should avoid it

Reply:
Yup, `lerna publish` does bumping by default.

Reply:
Yep just did a dry run, it already bumps internal used packages as well. We can remove this note